docker/scripts/test-volume-optimization.sh

```
@@ -0,0 +1,105 @@
+echo
+
+# 9. Final volume size
+echo "9. Final volume size after user package install..."
+VOLUME_SIZE=$(docker system df -v | grep docker_dev8-home | awk '{print $3}')
+echo -e "   docker_dev8-home: ${GREEN}${VOLUME_SIZE}${NC}"
+echo
+
+# 10. Compare with target
+echo "=== Results Summary ==="
+echo
+TARGET_SIZE=100  # 100 MB target
```

Shellcheck (0.11.0)

[warning] TARGET_SIZE appears unused. Verify use (or export if used externally).

(SC2034)